apps/api/src/app/events/usecases/digest-filter-steps/digest-filter-steps.usecase.ts

```diff
@@ -42,7 +42,7 @@ export class DigestFilterSteps {
         _creatorId: command.userId,
         type: StepTypeEnum.TRIGGER,
         content: '',
-      } as any,
+      } as MessageTemplateEntity,
```

Comment:
Just to remove the `any` and the warning. This object misses 2 properties to not raise the warn.